target/linux/ar71xx/image/tp-link.mk

```diff
@@ -510,6 +510,21 @@ define Device/tl-wa850re-v2
   MTDPARTS := spi0.0:128k(u-boot)ro,1344k(kernel),2304k(rootfs),256k(config)ro,64k(art)ro,3648k@0x20000(firmware)
 endef
 
+define Device/tl-wa855re-v1
+  $(Device/tplink)
+  DEVICE_TITLE := TP-LINK TL-WA855RE v1
+  BOARDNAME := TL-WA855RE-v1
+  DEVICE_PROFILE := TLWA855RE
+  TPLINK_HWID := 0x08550001
+	TPLINK_BOARD_NAME := TLWA855REV1
+	KERNEL := kernel-bin | patch-cmdline | lzma | mktplinkfw-kernel
+  IMAGES := sysupgrade.bin factory.bin
+  IMAGE/sysupgrade.bin := append-rootfs | tplink-safeloader sysupgrade
+  IMAGE/factory.bin := append-rootfs | tplink-safeloader factory
+  MTDPARTS := spi0.0:128k(u-boot)ro,1344k(kernel),2304k(rootfs),256k(config)ro,64k(art)ro,3648k@0x20000(firmware)
+endef
+TARGET_DEVICES += tl-wa855re-v1
```

Comment:
Please, move it below, where other TL-WAxxx definitions are included in `TARGET_DEVICES`.